jobs/integr8ly/ocp4/ocp4-testing-executor.yaml

```diff
@@ -0,0 +1,225 @@
+          name: INSTALLATION_TYPE
+          description: "Installation type (workshop for PDS clusters, managed otherwise"
+          choices:
+            - workshop
+            - managed
+      - string:
+          name: NAMESPACE_PREFIX
+          description: "Prefix of Integreatly Operator namespace, defauls to `BUILD_NUMBER`"
+      - bool:
+          name: CLEANUP_BEFOREHAND
+          default: true
+          description: "Indicates whether to cleanup before execution"
+      - bool:
+          name: CLEANUP_AFTERWARD
+          default: true
+          description: "Indicates whether to cleanup after execution"
+      - string:
+          name: NAMESPACE_CLEANUP_BEFOREHAND
+          description: "Namespace to be removed before the Integreatly Operator deploy. Can be left empty"
+      - string:
+          name: GH_CLIENT_ID
+          description: "GitHub Client ID for OAuth Apps, required for some of the walkthroughs"
+      - string:
+          name: GH_CLIENT_SECRET
+          description: "GitHub Client Secret for OAuth Apps, required for some of the walkthroughs"
+      - string:
+          name: TEST_SUITES_REPOSITORY
+          default: 'https://gitlab.cee.redhat.com/integreatly-qe/integreatly-qe.git'
+          description: "Repository of Integreatly test suites, YAML templates are stored there"
+      - string:
+          name: TEST_SUITES_BRANCH
+          default: 'master'
+          description: "Branch of Integreatly test suites repository"
+      - string:
+          name: RECIPIENTS
+          default: 'integreatly-qe@redhat.com'
+          description: "Whitespace- or comma-separated list of recipient addresses"
+      - choice:
+          name: 'AWS_REGION'
+          choices:
+            - 'eu-west-1'
+            - 'eu-west-2'
+            - 'us-east-1'
+          description: 'The AWS Region the cluster will be created in (openshift-install specific, e.g. non-pds)'
+          default: 'eu-west-1'
+      - string:
+          name: 'CREDENTIALS_ID'
+          default: 'iam-openshift4-credentials'
+          description: 'AWS IAM credentials used to create OpenShift cluster (openshift-install specific, e.g. non-pds)'
+      - choice:
+          name: 'COMPUTE_REPLICAS'
+          choices:
+            - '2'
+            - '3'
+          description: 'Number of compute replicas for the Openshift 4 cluster (openshift-install specific, e.g. non-pds)'
+          default: '2'
+      - choice:
+          name: 'CONTROL_PLANE_REPLICAS'
+          choices:
+            - '1'
+            - '2'
+            - '3'
+          description: 'Number of control plane replicas for the Openshift 4 cluster (openshift-install specific, e.g. non-pds)'
+          default: '1'
+      - choice:
+          name: TO_DO
+          description: "It specifies what stages of the pipeline will be executed. Provision and deprovision are for non-pds clusters only"
+          choices:
+            - provision + install + deprovision
```

Comment:
Would it be useful to also have `provision + install`?

Reply:
yes. I just put the initial array of options, which will be expanded later anyway (e.g. to include tests). Will add this option and merge. Thanks